index.html

```diff
@@ -173,155 +181,311 @@
-            </dd>
-          </dl>
+          <ol>
+            <li link-for="VideoMonitorEvent">If <var>e</var> is
+            <a>videomonitor</a>, create a <a>VideoMonitorEvent</a>, initialize
+            it to have a given name <var>e</var>, to not bubble, to not be
+            cancelable.
+            </li>
+            <li link-for="VideoProcessorEvent">If <var>e</var> is
+            <a>videoprocess</a>, create a <a>VideoProcessorEvent</a>,
+            initialize it to have a given name <var>e</var>, to not bubble, to
+            not be cancelable.
+            </li>
+            <li link-for="VideoMonitorEvent">Initialize the <a>trackId</a>
+            attribute to the value of the <code>id</code> attribute of the <a>
+              video worker source</a>.
+            </li><!-- TODO -->
+            <li link-for="VideoMonitorEvent">Initialize the <a>playbackTime</a>
+            attribute to the value of the <code>currentTime</code> attribute of
+            the <a>MediaStream</a> that contains the <a>video worker
+            source</a>.
+            </li>
+            <li link-for="VideoMonitorEvent">Initialize the
+            <a>inputImageBitmap</a> attribute to the bitmap data of the
+            <a>video worker source</a>'s video frame at the current stream
+            position at <a>playbackTime</a>.
+            </li>
+            <li>
+              <!-- TODO: clarify the definition of targets -->
+              Let <var>associated video workers</var> be the list of
+              <a data-lt="video worker">video workers</a> that share the same
+              <a>video worker source</a>.
+            </li>
+            <li>If <var>e</var> is <a>videomonitor</a>, dispatch the newly
+            created <a>VideoMonitorEvent</a> object at each <var>target</var>.
+            </li>
+            <li link-for="VideoProcessorEvent">If <var>e</var> is
+            <a>videoprocess</a>, initialize the <a>outputImageBitmap</a>
+            attribute to null, and dispatch the newly created
+            <a>VideoProcessorEvent</a> object at each <var>target</var>.
+            </li>
+          </ol>
         </section>
-        <section>
-          <h2>
-            <code id = event-videomonitorevent>VideoMonitorEvent</code> interface
-          </h2>
+      </section>
+      <section>
+        <h2>
+          <code id="event-videomonitorevent">VideoMonitorEvent</code> interface
+        </h2>
+        <p>
+          <dfn>Video monitor events</dfn> use the <a>VideoMonitorEvent</a>
+          interface for their <a>videomonitor</a> events:
+        </p>
+        <pre class="idl">
+          [Constructor(DOMString type, optional VideoMonitorEventInit videoMonitorEventInitDict), Exposed=Worker]
+          interface VideoMonitorEvent : Event {
+              readonly    attribute DOMString   trackId;
+              readonly    attribute double      playbackTime;
+              readonly    attribute ImageBitmap? inputImageBitmap;
```

Comment:
Thanks for your explanation and I am quite ok with it now.  

> Is that how you intended this to behave? Let me know, and I'll update the PR accordingly.

Yes, I think it is but I would like to leave this question open for @ChiahungTai to comment on. 

@anssiko, just let you know that I am out of office from 9/26 to 10/11, however, I will check messages occasionally, so sorry for my late reply during this period.